components/WysiwygContent.vue

```diff
@@ -38,3 +38,50 @@ export default {
   }
 }
 </script>
+
+<style>
+.ql-editor blockquote, .ql-editor h1, .ql-editor h2, .ql-editor h3, .ql-editor h4, .ql-editor h5, .ql-editor h6, .ql-editor ol, .ql-editor p, .ql-editor pre, .ql-editor ul {
+  counter-reset: list-1 list-2 list-3 list-4 list-5 list-6 list-7 list-8 list-9;
+  margin: 0;
+  padding: 0;
+}
+.ql-editor ol, .ql-editor ul {
+  padding-left: 1.5em;
+}
+.ql-editor ul>li:before {
+  content: "\2022";
+}
+.ql-editor ol li:before {
+  content: counter(list-0, decimal) ". ";
```

Comment:
are these generic for quill1 ?

Reply:
Yes, these are coming directly from quill 1 css